deploy/ansible/roles/k8s_resources/templates/prometheus.yml.j2

```diff
@@ -2,6 +2,12 @@ grafana:
   ingress:
     enabled: true
     fqdn: kube-grafana.{{ base_domain }}
+{% if dex.enabled == true %}
+
+    annotations:
+      nginx.ingress.kubernetes.io/auth-signin: https://auth.{{ cluster_name }}/oauth2/start?rd=https://$host$request_uri$is_args$args
+      nginx.ingress.kubernetes.io/auth-url: http://oauth2-proxy.kube-system.svc.cluster.local:4180/oauth2/auth
```

Comment:
Maybe it would be better to move `auth-signin` and `auth-url` values to variable and use it in all templates?

Reply:
not sure

Reply:
Please, do not do it. Component auth staff should be in comp's config.